docker-compose.yml

```diff
@@ -67,8 +67,18 @@ services:
       '--listen', '0.0.0.0:8000',
       '--max-third-party-nodes', '100000',
       '--update-every', '1',
+      '--metadata-path', '/metadata/metadata.json',
     ]
+    volumes:
+      - telemetry-core-metadata:/metadata
     ports:
       - 127.0.0.1:8000:8000
     expose:
       - 8000
+
+volumes:
+  # Just create it with:
+  # $ docker volume create telemetry-core-metadata
+  # $ docker run -it -v telemetry-core-metadata:/m alpine chown 1000:1000 /m 
+  telemetry-core-metadata:
+    external: true
```

Comment:
I'd say remove `external: true` and let it be managed by Docker Compose, much easier that way.
Then you can set permissions in Dockerfile and avoid manual actions, here is an example: https://github.com/subspace/subspace/blob/7b1412208889c3bf52aace9ea3d56dc328645eba/Dockerfile-node#L65-L67